src/schema/v2/artwork/index.ts

```diff
@@ -739,7 +739,7 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           const { price_paid_cents } = artwork
           const price_paid_currency = artwork.price_paid_currency || "USD"
           return {
-            cents: price_paid_cents,
+            cents: price_paid_cents || 0,
```

Comment:
does this mean we will return 0 in case where this is not specified on the artwork? I think if that is the case might be more expected to make it optional and return null

Reply:
I fixed this to allow null. The money class had some weird stuff about nulls. As long as we don't use `major` or `minor` in our queries, we'll be OK.

Reply:
Thanks!